src/vs/workbench/api/browser/mainThreadTerminalService.ts

```diff
@@ -129,7 +129,8 @@ export class MainThreadTerminalService implements MainThreadTerminalServiceShape
 			hideFromUser: launchConfig.hideFromUser,
 			isExtensionTerminal: launchConfig.isExtensionTerminal,
 			extHostTerminalId: extHostTerminalId,
-			isFeatureTerminal: launchConfig.isFeatureTerminal
+			isFeatureTerminal: launchConfig.isFeatureTerminal,
+			isExtensionOwnedTerminal: launchConfig.isExtensionOwnedTerminal
```

Comment:
We might want to rename "extension terminal" to "extension pty terminal" or something in the future, I struggle to come up with a good name to distinguish it clearly from regular terminals launched from extensions.